src/lib/js-source-map.ts

```diff
@@ -0,0 +1,45 @@
+import * as sourcemap from 'source-map'
+import {MappingItem} from 'source-map'
+
+type JavascriptSymbolMap = Map<number, Map<number, string>>
+
+// Looks like the d.ts description doens't properly define `initialize`
+// @ts-ignore
+sourcemap.SourceMapConsumer.initialize({
+  'lib/mappings.wasm': 'https://unpkg.com/source-map@0.7.3/lib/mappings.wasm',
```

Comment:
Woah this is pretty zany. I'm not sure what this is doing exactly, but it should be possible to pull this in as an import somehow? speedscope currently works offline (except that the fonts are different), so I'd like to keep it fully self-contained if possible (you can literally run speedscope from an extracted zip file with no internet connection).

Reply:
It seems like it should be possible to import the wasm module and have it be base64 encoded somehow. A tiny bit of information available here: https://github.com/mozilla/source-map/issues/408. It's possible that doing this will require mucking with the packaging config (I'm using https://parceljs.org/ at the moment, though have plans to switch to esbuild).